app/main/views/briefs.py

```diff
@@ -120,13 +122,40 @@ def update_brief_admin(brief_id):
     DEADLINES_TIME_OF_DAY = current_app.config['DEADLINES_TIME_OF_DAY']
     t = parse_time_of_day(DEADLINES_TIME_OF_DAY)
 
-    d = pendulum.parse(brief_json['clarification_questions_closed_at'])
-    combined = combine_date_and_time(d, t, DEADLINES_TZ_NAME)
-    brief.questions_closed_at = combined.in_timezone('UTC')
-
-    d = pendulum.parse(brief_json['applications_closed_at'])
-    combined = combine_date_and_time(d, t, DEADLINES_TZ_NAME)
-    brief.closed_at = combined.in_timezone('UTC')
+    if brief_json.get('add_user'):
```

Comment:
rather than including this in post, and just making this more complex, could you have endpoints for add/remove brief users?  e.g. POST/DELETE `/briefs/id/users/id`